pkg/executables/clusterctl.go

```diff
@@ -274,10 +278,57 @@ func (c *Clusterctl) buildConfig(clusterSpec *cluster.Spec, clusterName string,
 
 	return &clusterctlConfiguration{
 		configFile:               filePath,
-		bootstrapVersion:         fmt.Sprintf("kubeadm:%s", bundle.Bootstrap.Version),
+		bootstrapVersion:         fmt.Sprintf("%s:%s", kubeadmBootstrapProviderName, bundle.Bootstrap.Version),
 		controlPlaneVersion:      fmt.Sprintf("kubeadm:%s", bundle.ControlPlane.Version),
```

Comment:
Should we just change for these too?

Reply:
I didn't because I didn't need those values as constants
But if you feel it's better, I can do it